libs/application/ui-components/src/components/NationalIdWithName/NationalIdWithName.tsx

```diff
@@ -27,14 +26,23 @@ interface NationalIdWithNameProps {
   customId?: string
   customNationalIdLabel?: StaticText
   customNameLabel?: StaticText
+  phoneLabel?: StaticText
+  emailLabel?: StaticText
+  phoneRequired?: boolean
+  emailRequired?: boolean
   onNationalIdChange?: (s: string) => void
   onNameChange?: (s: string) => void
   nationalIdDefaultValue?: string
   nameDefaultValue?: string
+  phoneDefaultValue?: string
+  emailDefaultValue?: string
   errorMessage?: string
   minAgePerson?: number
   searchPersons?: boolean
   searchCompanies?: boolean
+  phone?: boolean
```

Comment:
Find the naming of phone and email slightly confusing, maybe something like showPhoneField or something a bit more descriptive 😅

Reply:
Done